packages/app/docker-entrypoint.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+
+# Error on bad command
+set -e
+
+# Defaults taken from .env.defaults in Medplum monorepo packages/app
+DEFAULT_MEDPLUM_BASE_URL=http://localhost:8103/
+DEFAULT_MEDPLUM_CLIENT_ID=
+DEFAULT_GOOGLE_CLIENT_ID=
+DEFAULT_RECAPTCHA_SITE_KEY=6LfHdsYdAAAAAC0uLnnRrDrhcXnziiUwKd8VtLNq
+DEFAULT_MEDPLUM_REGISTER_ENABLED=true
+DEFAULT_MEDPLUM_AWS_TEXTRACT_ENABLED=true
+
+# Find all JS files in the assets directory
+for file in $(find /usr/share/nginx/html/assets -name "*.js"); do
```

Comment:
This can be done in a single command, which will be a little faster and cleaner

See: https://github.com/medplum/medplum/blob/main/scripts/build-deb.sh#L270-L278

And you can avoid the `if`/`then`/`else` statements by just setting the default value:

```sh
${ENV_VAR:=default_value}
```